pkgs/development/interpreters/tcl/tcl-package-hook.sh

```diff
@@ -0,0 +1,74 @@
+# This hook ensures that we do the following in post-fixup:
+# * wrap any installed executables with a wrapper that configures TCLLIBPATH
+# * write a setup hook that extends the TCLLIBPATH of any anti-dependencies
+
+# Add a directory to TCLLIBPATH, provided that it exists
+_addToTclLibPath() {
+    local -r tclPkg="$1"
+    if [ -z "$tclPkg" ]; then
+        return
+    fi
```

Comment:
Please consistently use bashism or not.

Reply:
I've switched to using double-square brackets throughout. Does this address your concern, or was there another bashism/non-bashism that you think should be revisited?